### ISSUE TITLE
Describe relation between alpha argument and learning rate in Adam optimizer

### ISSUE THREAD

**Issue by niboshi** [chainer/chainer#4056]
Chainer [documents](https://docs.chainer.org/en/v4.0.0b1/reference/generated/chainer.optimizers.Adam.html) this argument as "step size", but other major frameworks describe it as "learning rate". ([pytorch](http://pytorch.org/docs/master/optim.html#torch.optim.Adam), [tensorflow](https://www.tensorflow.org/api_docs/python/tf/train/AdamOptimizer), [keras](https://keras.io/optimizers/#adam), [mxnet](https://mxnet.incubator.apache.org/api/python/optimization.html#mxnet.optimizer.Adam)).

The original paper describes it as "step size", so maybe we should leave the current description, but it may be better to mention "learning rate" as well.

**Comment by beam2d** [comment]
One complication is that `chainer.optimizers.Adam` also has `lr` property, which means "learning rate" and returns a *corrected version* of `alpha` used in the algorithm. I agree that we should mention how the `alpha` argument is called in other frameworks, but we need a careful explanation about the correspondence (I mean, we cannot just say "alpha is equivalent to learning rate").

**Comment by niboshi** [comment]
I see. I changed the title accordingly.

**Comment by niboshi** [comment]
bump

**Comment by toslunar** [comment]
I'd like to hide `Adam.lr` property, because it's not a meaningful value to me.  "Correcting" `alpha` is just computationally cheaper than correcting momentums.

**Comment by Crissman** [comment]
Merged.
